language-plutus-core/src/Language/PlutusCore/Constant/Typed.hs

```diff
@@ -38,6 +38,8 @@ module Language.PlutusCore.Constant.Typed
     , typedSubtractInteger
     , typedMultiplyInteger
     , typedDivideInteger
+    , typedQuotientInteger
+    , typedModInteger
```

Comment:
In the code `typedModInteger` goes after `typedRemainderInteger` and here it's the other way around.